src/DeepCopy/Filter/ObjectFilter.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace DeepCopy\Filter;
+
+/**
+ * Defines a filter that update the source object state only.
+ */
+interface ObjectFilter extends Filter
```

Comment:
How about `ChainableFilter` as to any filter implementing this interface will not stop the chain of filters, i.e. if there is a filter registered after this one it will be applied?